packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -401,31 +401,21 @@ export function getNetworkName(chainId: number) {
   return getBridgeUiConfigForChain(chainId).network.name
 }
 
-export function getSupportedNetworks(chainId = 0, includeTestnets = false) {
-  const testnetNetworks = [
-    ChainId.Goerli,
-    ChainId.ArbitrumGoerli,
-    ChainId.Sepolia,
-    ChainId.ArbitrumSepolia,
-    ChainId.StylusTestnet,
-    ...getOrbitChains({ mainnet: false, testnet: true }).map(
-      chain => chain.chainID
-    ),
-    ...getCustomChainsFromLocalStorage().map(chain => chain.chainID)
-  ]
-
-  const mainnetNetworks = [
-    ChainId.Ethereum,
-    ChainId.ArbitrumOne,
-    ChainId.ArbitrumNova,
-    ...getOrbitChains({ mainnet: true, testnet: false }).map(
-      chain => chain.chainID
-    )
-  ]
-
-  return isNetwork(chainId).isTestnet
-    ? [...mainnetNetworks, ...testnetNetworks]
-    : [...mainnetNetworks, ...(includeTestnets ? testnetNetworks : [])]
+export function getSupportedChainIds(
+  {
+    includeTestnets
+  }: {
+    includeTestnets: boolean
+  } = { includeTestnets: false }
+) {
+  return getChains()
+    .map(chain => chain.chainID)
```

Comment:
NIT: we don't really need a map here. You can either access `chain.chainID` on next line, or destructure: `.filter(({ chainID }) => {`

Reply:
As discussed offline, we return ChainIds here and I added the return type